best-practices/tagging/index.html

```diff
@@ -302,8 +302,9 @@ <h4>Headings</h4>
 				</aside>
 
 				<aside class="example" title="Multiline heading">
-					<pre><code class="html">&lt;h1>⠠⠉⠓⠁⠏⠞⠑⠗ ⠼⠛&lt;br/>
-⠠⠞⠓⠑ ⠠⠗⠑⠍⠁⠗⠅⠁⠃⠇⠑ ⠠⠞⠗⠥⠞⠓&lt;br/>
+					<p>Note that wbr is preferred over </p>
```

Comment:
I am not sure if this part of the change belongs in this PR. First of the sentence is not finished. It should probably say 
``` xml
Note that <code>&lt;wbr/></code> is preferred over <code>&lt;br/></code>.
```
Secondly, this change is not mentioned in the description of the PR, so I'm wondering if this was just accidental.

Reply:
I guess it belongs to https://github.com/daisy/ebraille/pull/255.